e2e/testsuite/testsuite.go

```diff
@@ -177,52 +177,60 @@ func (s *E2ETestSuite) SetupTest() {
 	s.SetupPath(ibc.DefaultClientOpts(), defaultChannelOpts(s.GetAllChains()))
 }
 
-// SetupPath creates a path between the chains using the provided client and channel options.
+// SetupPath creates paths between the all chains using the provided client and channel options.
 func (s *E2ETestSuite) SetupPath(clientOpts ibc.CreateClientOptions, channelOpts ibc.CreateChannelOptions) {
```

Comment:
I was tempted to rename this to `SetupPaths` so the method below could be named `SetupPath`. Would like feedback if we agree on that (and the addition of the method below, obviously) and I can do that in a separate PR.

Reply:
Yeah, it used to create only one path always, and still does whenever there are only two chains. But it now creates paths such that all chains are connected in a line A->B->C->etc.